<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QItemController</class>
 <widget class="QDialog" name="QItemController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ItemController v.1.0</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbl_modelName">
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>MODEL_NAME:---</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QDisplayLabel" name="lb_itemImg">
           <property name="minimumSize">
            <size>
             <width>450</width>
             <height>320</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>30</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;No Video Stream ...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDisplayLabel" name="lb_imgNormalized">
           <property name="minimumSize">
            <size>
             <width>450</width>
             <height>320</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-style:italic;&quot;&gt;Normalized Image&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableWidget" name="tabWidget">
           <property name="columnCount">
            <number>4</number>
           </property>
           <column>
            <property name="text">
             <string>SizeID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>ActualSizeInMilim</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>ModelErrInMilim</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string/>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_status">
           <property name="font">
            <font>
             <pointsize>40</pointsize>
            </font>
           </property>
           <property name="text">
            <string>STATUS:---</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pb_loadModel">
        <property name="text">
         <string>Load model</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Start capturing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_calibration">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Calibrate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_captureControl">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Control</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDisplayLabel</class>
   <extends>QLabel</extends>
   <header location="global">qdisplaylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
